LLMonFHIR/FHIR Display/ResourceChatView.swift

```diff
@@ -0,0 +1,71 @@
+//
+// This source file is part of the Stanford LLM on FHIR project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SpeziOpenAI
+import SwiftUI
+
+
+struct ResourceChatView: View {
```

Comment:
I would suggest to rename this to `OpenAIChatView`. It doesn't really have any knowledge about FHIR resources so this might be a more applicable name.